docs/user_guide/datasets/acs_public_coverage.rst

```diff
@@ -0,0 +1,274 @@
+   *  - DEYE
+      - Vision difficulty:
+         1. Yes
+         2. No
+         
+   *  - DREM
+      - Cognitive difficulty:
+         1. Yes
+         2. No
+         
+   *  - PINCP
+      - Total annual income per person as an integer between -19997 and 4209995
+        US dollars. Loss of $19998 or more is coded as -19998. Income of
+        $4209995 or more is coded as 4209995.
+
+   *  - ESR
+      - Employment status recode:
+         1. Civilian employed, at work
+         2. Civilian employed, with a job but not at work
+         3. Unemployed
+         4. Armed forces, at work
+         5. Armed forces, with a job but not at work
+         6. Not in labor force
+         
+   *  - ST
+      - State code: Please see data dictionary at `ACS PUMS documentation
+         <https://www.census.gov/programs-surveys/acs/microdata/documentation.2018.html>`_
+         for the full list of state codes.
+
+   *  - FER
+      - Gave birth to child within the past 12 months:
+         1. Yes
+         2. No
+
+   *  - RAC1P
+      - Race code
+         1. White alone
+         2. Black or African American alone
+         3. American Indian alone
+         4. Alaska Native alone
+         5. American Indian and Alaska native tribes specified; or American
+            Indian or Alaska Native, not specified and no other races
+         6. Asian alone
+         7. Native Hawaiian and Other Pacific Islander alone
+         8. Some Other Race alone
+         9. Two or More races
+
+
+The target label is given by PUBCOV, which can be used for a binary
+classification task.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 7 30
+   :stub-columns: 1
+
+   *  - Column name
+      - Description
+
+   *  - PUBCOV
+      - Public health coverage, with PUBCOV == 1 if the individual has public
+        health coverage, else 0
+
+
+.. _acs_public_coverage_discussion:
+
+Discussion
+^^^^^^^^^^
+
+A meaningful evaluation of algorithmic (un)fairness considers the context in
+which the machine learning model will be used. In this way, we can avoid
+falling into abstraction traps (see :footcite:`selbst2019fairness`). For
+example, the relationship between demographics and healthcare coverage may vary
+widely by state - a model that works well in one state may not be suitable to
+use in a different state.
+
+To investigate how viable ACSPublicCoverage is as a fairness benchmark dataset,
+we must first ask what real-world tasks might require a prediction of whether
+an individual is covered by public health insurance. Usually, the entity that
+needs this information (e.g., the government checking for eligibility in a
+social program) will simply ask the individual directly. Ding et al. do not
+provide any examples, so it is difficult to evaluate what notion of fairness is
+applicable without further context. 
+
+The remainder of this discussion focuses on preliminary data analysis of
+ACSPublicCoverage and identifies potential problems that users should be aware
```

Comment:
Who are the "users" here, and what are they using it for? People (e.g., data scientists?) who might use this dataset for fairness benchmarks? That may be worth calling out explicitly.

Reply:
I think the main audience would be either researchers who want to benchmark some fair-ml approach, or people who want to use the dataset as an educational resource for educators/blog post writers who are looking for a dataset to use in examples/tutorials/etc. 

I have attempted to rewrite the piece a bit, but I am not sure whether everything I have written is correct. It would be great if some of our US citizens could check whether my interpretation is correct.

```
Another potential issue that researchers and educators should be aware 
of before using this dataset as a benchmark or educational resource, is the
construct validity of the measurements (see also :ref:`construct_validity`).
The purpose of ACSPublicCoverage is to "predict whether a low-income 
individual, not eligible for Medicare, has coverage from public health 
insurance".
Here, Ding et al. define 'low-income' as individuals whose income is 
below $30.000.
This income threshold `was calculated
<https://github.com/zykls/folktables/issues/14>`_ 
approximately in accordance with Affordable Care Act (ACA) eligibility 
guidelines, qualifying adults with income up to 133% of the 2021 Federal 
Poverty Line for Medicaid.
An important question is to what extent the $30.000 threshold is a valid
measurement of low income, in relation to public health insurance 
eligibility.
Firstly, the dataset contains data from 2018, in which the federal poverty 
line was slightly different compared to 2021.
More importantly, however, the federal poverty line is the same across
all states, whereas the low-income threshold used to determine 
eligibility for Medicaid can differ per state.
In particular, not all states expanded Medicaid under the ACA. 
Consequently, some states have a much lower low-income threshold 
compared to the federal poverty line.
Additionally, the dataset concerns individuals, but low-income status also
depends on household size, which is not one of the 19 features in 
ACSPublicCoverage.
We observe a similar issue with the eligibility for Medicare, which 
is implemented through the exclusion of individuals of 65 years and older.
While 65 traditionally is the eligibilty age for Medicare, younger 
people with disabilities may still qualify :footcite:`usgov2014who`.
Depending on how the predictions are used, the mismeasurement of 
low income and Medicare eligibility may result in allocation harms 
based on state or disability status (see also :ref:`types_of_harms`).
```

From this page, it looks like Medicaid eligibility may differ per state: https://www.healthcare.gov/lower-costs/